src/script/sign.cpp

```diff
@@ -267,6 +267,10 @@ class SignatureExtractorChecker final : public BaseSignatureChecker
         }
         return false;
     }
+    // These methods just pass through to checker
+    bool CheckSchnorrSignature(Span<const unsigned char> sig, Span<const unsigned char> pubkey, SigVersion sigversion, const ScriptExecutionData& execdata, ScriptError* serror = nullptr) const { return checker.CheckSchnorrSignature(sig, pubkey, sigversion, execdata, serror); }
```

Comment:
```
/usr/bin/ccache g++ -m64 -std=c++17 -DHAVE_CONFIG_H -I. -I../src/config   -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -I. -I./secp256k1/include  -DBOOST_SP_USE_STD_ATOMIC -DBOOST_AC_USE_STD_ATOMIC -isystem /tmp/cirrus-ci-build/depends/x86_64-pc-linux-gnu/include -I./leveldb/include -I./leveldb/helpers/memenv -I./univalue/include -I/tmp/cirrus-ci-build/depends/x86_64-pc-linux-gnu/include/  -DHAVE_BUILD_INFO -D__STDC_FORMAT_MACROS -DPROVIDE_MAIN_FUNCTION  -fstack-reuse=none -Wstack-protector -fstack-protector-all -fcf-protection=full -fstack-clash-protection   -Werror=vla -Werror=switch -Werror=unused-variable -Werror=date-time -Werror=return-type -Werror=sign-compare -Werror=suggest-override   -fPIE -pipe -O2  -fno-extended-identifiers -c -o script/libbitcoin_common_a-sign.o `test -f 'script/sign.cpp' || echo './'`script/sign.cpp
script/sign.cpp:271:10: error: ‘virtual bool {anonymous}::SignatureExtractorChecker::CheckSchnorrSignature(Span<const unsigned char>, Span<const unsigned char>, SigVersion, const ScriptExecutionData&, ScriptError*) const’ can be marked override [-Werror=suggest-override]
  271 |     bool CheckSchnorrSignature(Span<const unsigned char> sig, Span<const unsigned char> pubkey, SigVersion sigversion, const ScriptExecutionData& execdata, ScriptError* serror = nullptr) const { return checker.CheckSchnorrSignature(sig, pubkey, sigversion, execdata, serror); }
      |          ^~~~~~~~~~~~~~~~~~~~~
cc1plus: some warnings being treated as errors

Reply:
Changed the approach entirely, shouldn't be a problem.